.clusterfuzzlite/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# -----------------------------------------------------------------------------
+# SPDX-FileCopyrightText: Copyright © 2024 bomctl a Series of LF Projects, LLC
+# SPDX-FileName: .clusterfuzzlite/DOCKERFILE
+# SPDX-FileType: SOURCE
+# SPDX-License-Identifier: Apache-2.0
+# -----------------------------------------------------------------------------
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+FROM gcr.io/oss-fuzz-base/base-builder-go
+
+ARG APT_PKGS="make autoconf automake libtool"
+
+RUN apt-get update \
+  && apt-get install --yes --no-install-recommends ${APT_PKGS} \
+  && git clone --depth 1 https://github.com/bomctl/bomctl.git
```

Comment:
Actually if this is used in CI only, can we just mount `$GITHUB_WORKSPACE` (the already cloned repo in the runner) as a volume or copy it in?

Reply:
Sorry, I don't think mounting a volume during a build is a thing. Maybe just try `COPY . bomctl/` or something? Assuming the build context is the runner's workspace directory, and you didn't `cd` to another directory